frontend/discourse/app/controllers/signup.js

```diff
@@ -542,6 +544,33 @@ export default class SignupPageController extends Controller {
     this.login.externalLogin(provider, { signup: true });
   }
 
+  @action
+  goToLogin() {
+    let loginName;
+
+    const email =
+      this.accountEmail ||
+      document.getElementById("new-account-email")?.value ||
```

Comment:
I'm curious why we need to fallback to querying the value of the element directly when the value of the input field is tracked by the `accountEmail` property?